04_documentation/markup_languages_slides.md

```diff
@@ -0,0 +1,254 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Source Code Pro';
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# Lightweight Markup Languages
+
+---
+
+## Learning Goals
+
+- You know common markup languages.
+- You know important parts of the syntax of these languages.
+- You know what benefits these languages have.
+- You know how to convert files written in these languages.
```

Comment:
I think it would be good to name the markup languages here already and to distinguish whether students learn general things about markup languages or specific things about md, rst, and yml.

Reply:
Fixed.